project/build.scala

```diff
@@ -32,7 +32,7 @@ object build extends Build {
     tokens,
     transversers,
     trees
-  )
+  ) dependsOn (scalameta)
```

Comment:
@fommil What would you advise? I'm kinda scared of dependsOn. Maybe we should do `console := (console in scalameta in Compile).value` instead?